src/libraries/System.Security.Cryptography.OpenSsl/src/System.Security.Cryptography.OpenSsl.csproj

```diff
@@ -12,7 +12,7 @@
     the facades when the package is restored. -->
     <PackageTargetFramework Condition="$(TargetFramework.StartsWith('net4'))">net461</PackageTargetFramework>
   </PropertyGroup>
-  <PropertyGroup Condition="'$(TargetsUnix)' != 'true' or '$(TargetsBrowser)' == 'true'">
+  <PropertyGroup Condition="'$(TargetsUnix)' != 'true'">
```

Comment:
I'm not sure I understand this change. OpenSsl isn't supported on Browser, so we still should be generating PNSE.

Reply:
```TargetsBrowser == true``` => ```TargetsUnix != true```,  the second part of the condition is redundant.

Reply:
Ah, got it. I read the first part wrong. Thanks.